book/src/format.md

```diff
@@ -29,6 +29,17 @@ Like built-in derives (e.g. `#[derive(Debug)]`), `#[derive(Format)]` will add `F
 
 > ⚠️ Do *not* use the API used by the expansion of the `derive(Format)` macro; it is *unstable*.
 
+By default the derive assumes `defmt` to exist in your crate's extern prelude.
+If that is not the case you can overwrite the crate path the derive should use in its expansion via the `defmt(crate = path)` helper attribute.
+
+```rust
+extern crate defmt as other_defmt;
```

Comment:
Why is this `extern` and not `use`?

```suggestion
use defmt as other_defmt;
```

Reply:
Either works, I figured this is clearer for the case where the crate has a different name (usually implied via the Cargo.toml dependency entry)

Reply:
Ah. And when you do it via the cargo toml you have to use `extern crate`?

Reply:
No, its more that `use foo as bar` can be used for anything thats called `foo` in the current scope. `extern crate foo as bar` only renames a crate called foo in the dependencies of the current crate.